app/commands/build_response_command.rb

```diff
@@ -64,7 +67,7 @@ def apply_root_attributes(input_data, to:)
   def validate_office_code_in_case_number
     return if case_number.nil? || office_present?
 
-    errors.add(:case_number, :invalid_office_code, command: command_name, uuid: uuid)
+    errors.add(:case_number, :invalid_office_code)
```

Comment:
I have simplified the error reporting code so that it does not need the command or the uuid anymore